src/main/java/homework_1/Main.java

```diff
@@ -1,9 +1,22 @@
 package homework_1;
 
-public class Main {
-
-  public static void main(String[] args) {
-    System.out.println("Hello homework!");
-  }
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
 
-}
+public class Main {
+    public static void main(String[] args) {
+        if (Charset.defaultCharset() != StandardCharsets.UTF_8) {
+            System.out.println("For correct display cyrillic args need to run with \"-Dfile.encoding=\"UTF-8\"\"");
+        }
+        if (args.length == 0) {
+            System.out.println("No arguments provided!");
+        } else for (String arg : args) {
```

Comment:
Opt: you forgot brackets after else. So for is IN, but sout is OUT

Reply:
added brackets